src/test/java/org/gridsuite/filter/server/GenerateScriptFromFiltersTest.java

```diff
@@ -44,6 +47,8 @@ public void generateScriptLineFilterTest() {
                 "      (FiltersUtils.matchID('lineId1', equipment) || FiltersUtils.matchName('lineName1', equipment))\n" +
                 "      && FiltersUtils.isLocatedIn(['ES','FR'], equipment.terminal1)\n" +
                 "      && FiltersUtils.isLocatedIn(['IT','PT'], equipment.terminal2)\n" +
+                "      && FiltersUtils.matchesFreeProps(['region':['north']], equipment.terminal1))\n" +
+                "      && FiltersUtils.matchesFreeProps(['region':['south']], equipment.terminal2))\n" +
                 "      && FiltersUtils.isRangeNominalVoltage(equipment.terminal1, 225.0, 250.0)\n" +
```

Comment:
why there is more spaces after double cote "

Reply:
I just used the same formatting as lines around.